lib/package.json

```diff
@@ -43,15 +43,11 @@
     "luxon": "^1.0.0"
   },
   "dependencies": {
-    "@babel/preset-typescript": "^7.1.0",
-    "@babel/runtime": "^7.0.0",
```

Comment:
Bring babel runtime back. It's direct dependency.

Reply:
runtime dependency?  will try

Reply:
By the way babel runtime is not used anymore